### ISSUE TITLE
Linking the deep Pacific Ocean and surface Southern Ocean 

### ISSUE THREAD

**Issue by gmacgilchrist** [abby-baskind/seniorthesis#6]
To gain some insight about how the conditions in the Pacific could be influencing carbon exchange in the Southern Ocean, we need to determine how waters in these two regions are linked.

The common approach to doing this is to consider the properties of the water that are materially conserved (total quantities remain constant in the presence of mixing), which are notionally temperature and salinity. This is the concept of identifying "water masses" and assessing their changing properties over geographical space. The combination of temperature and salinity is the water density, and analysis is often done on constant density surfaces, with these surface generally representing water masses. Unfortunately, density is not a materially conserved quantity, so requires careful treatment when considering properties across large geographical areas and at different pressures (as we are doing here). There is a quantity, called *neutral density* that seeks to minimize the problems associated with this - it approximates the "neutral surface" along which water masses move. However, it is not straightforward to calculate.

I'm opening this issue simply to have a space to discuss different ways that one could considering linking the water masses in these regions, and have the capacity to do so consistently across the model suite. 

Some options include (in rough order of increasing complexity):
- carefully considered potential density
- temperature-salinity space
- just salinity (if the broad region is temperature stratified, then salinity could be sufficient to distinguish between water masses of interest)
- neutral density (@jbusecke do you know if anyone has calculated neutral density on Pangeo?)
- another biogeochemical tracer, e.g. P*
- other neutral surfaces such as those considered [here](https://github.com/geoffstanley/neutral-surfaces)

**Comment by jbusecke** [comment]
> neutral density (@jbusecke do you know if anyone has calculated neutral density on Pangeo?)

Nope, but if there are implementations in python we can make them more efficient with dask. I have used [fastjmd](https://github.com/xgcm/fastjmd95) successfully if potential density would suffice for a first go.
